### ISSUE TITLE
Footer mode does not stretch iframe card to bottom of screen

### ISSUE THREAD

**Issue by blhoward2** [maykar/custom-header#258]
When displaying a website in a full-screen iframe, using compact mode and footer mode, the iframe draws to where it would if compact mode was not on and the space was left blank. When footer mode is not on the card correctly fills the screen. When compact mode is turned off everything looks as it should. Verified in both Chrome and Fully Kiosk Browser. Using the latest stable home assistant and the latest custom-header.

Raw configuration:

```
custom_header:
  compact_mode: true
  disable_sidebar: true
  exceptions: []
  shadow: false
  footer_mode: true
views:
  - background-image: center / cover no-repeat fixed url('/local/blue-abstract.jpg')
    badges: []
    cards:
      - aspect_ratio: 50%
        style: |
          ha-card {
            height: 100%
            width: 100%
          }
        type: iframe
        url: 'http://192.169.0.123/'
    icon: 'mdi:home-outline'
    panel: true

```

![Screen Shot 2020-08-27 at 11 51 45 PM](https://user-images.githubusercontent.com/33612110/91519377-75722080-e8c0-11ea-892e-02c62fd0277f.png)

**Comment by maykar** [comment]
In the next update I will fix the extra padding at the bottom when using panel mode with footer mode. I noticed in your screenshots from the other issue you posted on you were using a transparent header background, this will result in something like the screenshot below:

![image](https://user-images.githubusercontent.com/25127328/91644930-cee86580-ea0e-11ea-89b8-2586452db5e3.png)

The grey portion is the iFrame and the header background is transparent, showing the views background behind it. This is by design to avoid overlap, but if you wanted to extend the iFrame completely behind the header you can use this advanced styling config:

`panel_view_css: 'padding-bottom: 0px;'`

I will close this issue when the next version is released, either today or tomorrow.

**Comment by blhoward2** [comment]
Thanks!

**Comment by maykar** [comment]
Sorry, didn't make it into the most recent release. Will be in next one for sure, CH needed a quick fix for the next release of HA and wanted to get it out there before people started upgrading HA.

**Comment by maykar** [comment]
Apologize for the lateness, but this has been fixed in 1.7.7
